test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -482,82 +106,90 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
       });
     });
 
-    it.skip('TODO: should exclude default plugins localPath', () => {
+    it('should exclude default plugins localPath', () => {
       // Confirm ".serverless-plugins/index.js" is not packaged
-      //
+      expect(fnIndividualZippedFiles).to.not.include('.serverless-plugins/index.js');
       // Replaces
       // https://github.com/serverless/serverless/blob/b12d565ea0ad588445fb120e049db157afc7bf37/test/unit/lib/plugins/package/lib/packageService.test.js#L87-L96
```

Comment:
It would be great to remove all these `// Replaces ...` references from implemented tests as they're not really needed anymore and were supposed to serve as guidance when implementing the new ones

Reply:
Done.

Reply:
Did you push these changes? I still can see them in all implemented test cases

Reply:
Sorry, should be ok now.